workshops/04-Gem-Packaging-Workshop/README.md

```diff
@@ -0,0 +1,81 @@
+# Gem Packaging
+
+In Ruby we use Gems to distribute reusable components.
+
+For example, `rspec` is distributed as a Gem.
+
+When we install a gem by invoking `gem install rspec` we download a package from RubyGems.org.
```

Comment:
This language might imply that you have to go to rubygems.org and sort it out yourself.

Maybe:
> When we install a gem by invoking `gem install rspec`, it'll download and install a package (from RubyGems.org) for you.